plugins/node/opentelemetry-instrumentation-redis-4/package.json

```diff
@@ -7,12 +7,12 @@
   "repository": "open-telemetry/opentelemetry-js-contrib",
   "scripts": {
     "test": "nyc ts-mocha -p tsconfig.json --require '@opentelemetry/contrib-test-utils' 'test/redis.test.ts'",
-    "test:debug": "cross-env RUN_REDIS_TESTS_LOCAL=true ts-mocha --inspect-brk --no-timeouts -p tsconfig.json 'test/redis.test.ts'",
-    "test:local": "cross-env RUN_REDIS_TESTS_LOCAL=true npm run test",
-    "test:docker:run": "docker run --rm -d --name otel-redis -p 63790:6379 redis:alpine",
-    "test:docker:stop": "docker stop otel-redis",
+    "test-services:start": "cd ../../.. && npm run test-services:start redis",
+    "test-services:stop": "cd ../../.. && npm run test-services:stop redis",
+    "test:with-test-services": "NODE_OPTIONS='-r dotenv/config' DOTENV_CONFIG_PATH=../../../test/test-services.env npm test",
+    "test:debug": "NODE_OPTIONS='-r dotenv/config' DOTENV_CONFIG_PATH=../../../test/test-services.env ts-mocha --inspect-brk --no-timeouts -p tsconfig.json 'test/redis.test.ts'",
```

Comment:
We could use a [`.mocharc.cjs` file](https://mochajs.org/#configuring-mocha-nodejs) for config options instead of specifying it in the CLI command.

Reply:
I believe mocha config files are like eslint config files where you can have multiple with custom names and pass them in wherever you need them. https://mochajs.org/#custom-locations

Reply:
Given that the diff for this part of this PR is not changing how ts-mocha is called:

```
-    "test:debug": "cross-env RUN_REDIS_TESTS_LOCAL=true ts-mocha --inspect-brk --no-timeouts -p tsconfig.json 'test/redis.test.ts'",
+    "test:debug": "NODE_OPTIONS='-r dotenv/config' DOTENV_CONFIG_PATH=../../../test/test-services.env ts-mocha --inspect-brk --no-timeouts -p tsconfig.json 'test/redis.test.ts'",
```

I think changing to .mocharc.cjs is out of scope for this change and would distract from the test-services-related changes being made. Thoughts?

Reply:
Yep that's fair. I was mostly pointing out because of the long npm scripts in general that you mentioned, but otherwise agree it doesn't need to happen here.